src/mjolnir/convert_transit.cc

```diff
@@ -569,7 +558,6 @@ void AddToGraph(GraphTileBuilder& tilebuilder_transit,
     // Get the platform information
     GraphId platform_graphid = stop_edges.second.origin_pbf_graphid;
     const Transit_Node& platform = tile_pbf.nodes(platform_graphid.id());
-    const std::string& origin_id = platform.onestop_id();
```

Comment:
the next removed lines are all due to removing the 2 arguments to the `GetShape()` function